content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -210,11 +138,14 @@ spec:
       resources:
         limits:
           nvidia.com/gpu: 1
-  nodeSelector:
-    accelerator: nvidia-tesla-p100 # or nvidia-tesla-k80 etc.
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        – matchExpressions:
+          – key: beta.amd.com/gpu.family.RV # Raven GPU family
```

Comment:
Preview of updated page and this section: https://deploy-preview-36985--kubernetes-io-main-staging.netlify.app/docs/tasks/manage-gpus/scheduling-gpus/#node-labeller

nit: for a more a consistent flow of this section, the page has
`kubectl describe node cluster-node-23`
The output has a label of `beta.amd.com/gpu.family.AI=1`
Then shortly after this, we have a pod manifest with an affinity but to a different gpu family. 
Non-blocking suggestion is to match the gpu family
```suggestion
          – key: beta.amd.com/gpu.family.AI # Arctic Islands GPU family
```